### PR TITLE
[VEN-2882] Multichain Governance on Base Sepolia and Mainnet

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

File: simulations/vip-384/basesepolia.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+      OMNICHAIN_GOVERNANCE_EXECUTOR_ABI,
+      provider,
+    );
+    executorOwner = new ethers.Contract(OMNICHAIN_EXECUTOR_OWNER, OMNICHAIN_EXECUTOR_OWNER_ABI, provider);
+    lastProposalReceived = await executor.lastProposalReceived();
+    await pretendExecutingVip(await vip006());
+  });
+
+  describe("Pre-VIP behaviour", async () => {
+    it("Normal Timelock has default admin role on OP sepolia", async () => {
```
Comment (GitGuru7) [inline simulations/vip-384/basesepolia.ts:43]:
```suggestion
    it("Normal Timelock has default admin role on base sepolia", async () => {
```

File: vips/vip-384/bsctestnet.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        signature: "executeGrantPermissions(uint256)",
+        params: [1],
+        dstChainId: LzChainId.basesepolia,
+      },
+      {
+        target: ACM_AGGREGATOR,
+        signature: "executeGrantPermissions(uint256)",
+        params: [3],
+        dstChainId: LzChainId.basesepolia,
+      },
```
Comment (chechu) [inline vips/vip-384/bsctestnet.ts:58]: If we include this step (granting timelocks to execute privilege functions) here, I think we could also include the step to revoke permissions from the Guardian:
```
      {
        target: ACM_AGGREGATOR,
        signature: "executeRevokePermissions(uint256)",
        params: [2],
        dstChainId: LzChainId.basesepolia,
      },
```

That we are currently adding in the PR https://github.com/VenusProtocol/vips/pull/435

**Review by chechu — COMMENTED** [review]
Testnet commands lgtm

File: src/types.ts [hunk 3]
```diff
@@ -10,7 +10,8 @@ export type SUPPORTED_NETWORKS =
   | "zksyncsepolia"
   | "zksyncmainnet"
   | "opsepolia"
-  | "opmainnet";
+  | "opmainnet"
+  | "basemainnet";
```
Comment (chechu) [inline src/types.ts:14]:
```suggestion
    | "basesepolia"
    | "basemainnet";
```

**Comment by chechu** [comment]
This VIP should add commands to refund the provider of the bootstrap liquidity for Base mainnet:
* 5,000 USDC ([TX](https://basescan.org/tx/0x84117a3a66a63ba784c07a096b51901eeb7bb0b0ba38f686ad1ab591218730c3)) - to be refunded with 5,000 USDC from the Venus Treasury
* 2 WETH ([TX](https://basescan.org/tx/0x4d6fde66b30be733f643a3bebe60b102833160397bbfdeec1de1cdbcbf617c71)) - to be refunded with 2 ETH from the Venus Treasury
* 0.05 cbBTC ([TX](https://basescan.org/tx/0x7e3021c203b8430d014e678094c59b4e3f7e00966cb241e1c684080c56bcba2c)) - to be refunded with 0.05 BTCB from the Venus Treasury

Destination address: TBD

**Comment by GitGuru7** [comment]
Mainnet Multichain VIP LGTM

**Review by chechu — COMMENTED** [review]
Omnichain Governance commands, including the pre-loaded commands in the ACMCommandsAggregator, lgtm

**Review by kkirka — APPROVED** [review]
Commands and addresses LGTM. I also checked the permissions and they make sense.

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/408?chainId=56

**Pull request merged**